packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -404,6 +409,96 @@ export default class VTable {
         return this.getTd(this.row, this.col);
     }
 
+    highlightSelection(start: HTMLTableCellElement, end: HTMLTableCellElement) {

```

Comment:
same here, use table cell indexes (row, col) instead of the actual TD element.

Because table cell can be merged. Consider:
+--------+---------+---------+
|  cell 1  |  cell 2   |  cell 3 |
+-------+---------+--------+
| cell 4  | cell 5 merge 6 |
+-------+---------------+

Since cell 5 and 6 are merged, they share the same TD. when we specify that TD, it is hard to tell if cell 3 should be selected.

Reply:
The top benefit of using VTable is to allow you forget about TD, but always use index. TD is only used when writeBack() is called.